packages/operator/pkg/config/deployment.go

```diff
@@ -52,6 +52,15 @@ type ModelDeploymentSecurityConfig struct {
 	RoleName string `json:"roleName"`
 }
 
+
+type Toleration struct {
```

Comment:
Why we can't use the `corev1.Toleration` struct here?

Reply:
yep, replaced